styles/base.css

```diff
@@ -106,20 +120,34 @@ h4,
 
 /*#endregion */
 
-/* Headlines */
+/* Headlines + Non-headline type */
 
 :root {
     --h1-font-size: calc(40rem / 16);
     --h1-line-height: 120%;
     --h2-font-size: calc(32rem / 16);
     --h3-font-size: calc(24rem / 16);
```

Comment:
The renaming of these H1, H2, H3 variables to XL, LG, MD will come in a future, separate PR.